LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2019-2020 David Ralph (ohlookitsderpy)
+Copyright (c) 2019-2020 David Ralph (ohlookitsderpy) and spinfish
```

Comment:
Please don't touch the license file. I'd prefer it if you included credits in the README file or somewhere in the plugin itself (as long as it isn't too intrusive)

Reply:
👍